sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -589,9 +617,8 @@ public void startBundle() {
       public void processElement(ProcessContext context) throws Exception {
         T record = context.element();
 
-        preparedStatement.clearParameters();
-        spec.getPreparedStatementSetter().setParameters(record, preparedStatement);
-        preparedStatement.addBatch();
+        records.add(record);
+        processRecord(record);
```

Comment:
Any reason why `processRecord` is needed here? Can we just do it in one place, in `executeBatchWithBackoff`?